### PR TITLE
Disallow bots on Wikilink

### PR DESCRIPTION
**Pull request by katydidnot**
Bug: T368660

## Description
Describe your changes in detail following the [commit message guidelines](https://www.mediawiki.org/wiki/Gerrit/Commit_message_guidelines)

Attempting to Implement bot blocking in a different way using nginx configuration.

## Rationale
[//]: # (Why is this change required? What problem does it solve?)

Stop bots from scraping Wikilink csv files. 

## Phabricator Ticket
[//]: # (Link to the Phabricator ticket)

https://phabricator.wikimedia.org/T368660

## How Has This Been Tested?
[//]: # (- Did you add tests to your changes? Did you modify tests to accommodate your changes?)
[//]: # (- Can this change be tested manually? How?)

## Screenshots of your changes (if appropriate):
[//]: # (It can also be a GIF to prove that your changes are working)

## Types of changes
What types of changes does your code introduce? Add an `x` in all the boxes that apply:
- [X] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

### CODE REVIEW

**Review by suecarmol — CHANGES_REQUESTED** [review]
This looks good, but you should also create a robots.txt file. I found these guides on how to do it in Django. The file should also include all the files we listed in #370: 

https://adamj.eu/tech/2020/02/10/robots-txt/
https://learndjango.com/tutorials/add-robotstxt-django-website

**Comment by jsnshrmn** [comment]
> This looks good, but you should also create a robots.txt file. I found these guides on how to do it in Django. The file should also include all the files we listed in #370:
> 
> https://adamj.eu/tech/2020/02/10/robots-txt/ https://learndjango.com/tutorials/add-robotstxt-django-website

I would recommend just serving the robots.txt file out of nginx. I think that's the right layer to handle rate limiting and ua checks like this. That's where we handle things like this in our other projects.

What do you think @suecarmol? I don't want @katydidnot to have to vacillate between divergent review advice, so we can talk it through if needed.

**Comment by suecarmol** [comment]
> > This looks good, but you should also create a robots.txt file. I found these guides on how to do it in Django. The file should also include all the files we listed in #370:
> > https://adamj.eu/tech/2020/02/10/robots-txt/ https://learndjango.com/tutorials/add-robotstxt-django-website
> 
> I would recommend just serving the robots.txt file out of nginx. I think that's the right layer to handle rate limiting and ua checks like this. That's where we handle things like this in our other projects.
> 
> What do you think @suecarmol? I don't want @katydidnot to have to vacillate between divergent review advice, so we can talk it through if needed.

Yeah, we should discuss it in our next engineering meeting!

**Review by jsnshrmn — APPROVED** [review]
Looks good to me and does in fact serve up the /robots.txt file locally.

File: extlinks/urls.py [hunk 1]
```diff
@@ -2,6 +2,7 @@
 from django.contrib import admin
 from django.urls import include, path
 from django.conf import settings
+from django.views.generic import TemplateView
```
Comment (suecarmol) [inline extlinks/urls.py:5]: Let's delete this unused import.

**Review by suecarmol — COMMENTED** [review]
Just a teeny-tiny change is needed before we merge this!

File: extlinks/urls.py (hunk 1 above)
Comment (katydidnot) [reply extlinks/urls.py]: Oh good catch!

**Review by suecarmol — APPROVED** [review]
Thank you for working (and re-working) on this ticket!

**Pull request merged**